thrift/src/main/java/com/linecorp/armeria/common/thrift/text/StructContext.java

```diff
@@ -198,9 +200,18 @@ protected TField getTFieldByName(String name) throws TException {
                     // Workaround a bug where the generated 'FieldMetaData' does not provide
                     // a fully qualified class name.
                     final String fqcn = clazz.getPackage().getName() + '.' + elementMetaData.getTypedefName();
-                    try {
-                        classMap.put(fieldName, Class.forName(fqcn));
-                    } catch (ClassNotFoundException ignored) {
+                    Class<?> fieldClass = fieldMetaDataClassCache.get(fqcn);
+                    if (fieldClass == null) {
+                        fieldClass = fieldMetaDataClassCache.computeIfAbsent(fqcn, key -> {
+                            try {
+                                return Class.forName(fqcn);
```

Comment:
How about using [`Class#forName(, false ,)`](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/lang/Class.java#L448) not to initialize an instance? Initializing seems unnecessary by specifications. https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4 I guess, it makes lock shorter by reducing an overhead of class loader.